src/Core.UnitTests/Notifications/DisabledNotificationsStorageTests.cs

```diff
@@ -36,7 +36,10 @@ namespace SonarLint.VisualStudio.Core.UnitTests.Notifications
     [TestClass]
     public class DisabledNotificationsStorageTests
     {
-        private string ExpectedDisabledNotificationsFilePath => GetExpectedDisabledNotificationsFilePath();
+        private const string DefaultRootPath = "c:\\users\\someuser";
```

Comment:
Fixing up the existing tests so they pass. The existing tests are a bit messy, but refactoring them was beyond the scope of this commit.